sdk/src/crypto/cose/ocsp.rs

```diff
@@ -68,7 +73,36 @@ pub fn check_ocsp_status(
             OcspFetchPolicy::FetchAllowed => {
                 fetch_and_check_ocsp_response(sign1, data, ctp, tst_info, validation_log)
             }
-            OcspFetchPolicy::DoNotFetch => Ok(OcspResponse::default()),
+            OcspFetchPolicy::DoNotFetch => {
+                if let Some(ocsp_response_ders) = ocsp_responses {
```

Comment:
Add a comment here what we are doing.  

// If we are not fetching use the supplied OCSP responses.  If any not revoked found then that is a good status.  All other status are just informational.